rotkehlchen/api/rest.py

```diff
@@ -5521,3 +5521,19 @@ def fetch_token_balance_for_address(
             return wrap_in_fail_result(str(e), status_code=HTTPStatus.CONFLICT)
 
         return _wrap_in_ok_result(result=balance)
+
+    def get_registered_devices(self) -> Response:
```

Comment:
Also add an one liner docstring

```suggestion
    def get_premium_registered_devices(self) -> Response:
```